src/components/magnet/magnet-util.ts

```diff
@@ -12,16 +12,83 @@ export function getFieldMagnitudeAndDirection(magnets: PossibleMagnet[], x: numb
     return [0, 0];
   }
 
-  // FAKE: strongest at points at either end
-  const dx1 = (magnet.x + (magnet.length / 2)) - x;
-  const dx2 = (magnet.x - (magnet.length / 2)) - x;
-  const dy = magnet.y - y;
-
-  const dxMin = Math.min(Math.abs(dx1), Math.abs(dx2));
-  const distance = Math.min(Math.sqrt((dxMin * dxMin) + (dy * dy)), 150);
-  const magnitude = Math.max(0, (1 - (distance / 150)));
-
-  // FAKE: point at one end
-  const direction = Math.atan2(dy, dx1);
-  return [magnitude, direction];
+  const dx = x - magnet.x;
```

Comment:
Even though I got rid of all of this code for final calculations, I figured it was worth leaving as part of the Git history